scripts/mutation-triage.sh

```
@@ -0,0 +1,103 @@
+  local id="$1"
+  local file="$2"
+  local search="$3"
+  local replace="$4"
+  local test_cmd="$5"
+
+  local backup log_file patch_file
+  backup="$(mktemp)"
+  log_file="$(mktemp)"
+  patch_file="$(mktemp)"
+
+  trap "restore_backup '${backup}' '${file}'" EXIT INT TERM
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)